lib/checks/landmarks/landmark-is-unique.json

```diff
@@ -0,0 +1,16 @@
+{
+	"id": "landmark-is-unique",
+	"evaluate": "landmark-is-unique.js",
+	"after": "landmark-is-unique-after.js",
+	"tags": ["best-practice"],
+	"options": {
```

Comment:
What was the reason for this options object? I don't see this being used anywhere so it looks like this should be removed.

Reply:
Removed.